webroot/results/admin/scripts/remove_imported_data.php

```diff
@@ -19,4 +19,4 @@
 //     array('s', &$compId)
 //   );
 
-print "Removal attempted.";
+jsonReturn();
```

Comment:
So these messages ("Imported Results..." and "Removal attempted.") were never actually used anywhere?

Reply:
No. We don't use it.